File: week01_c/lecture01/calculator.c
#include <stdio.h>
#include <cs50.h>

int add(int a, int b);

int main(void)
{
    int x = get_int("x: ");
    int y = get_int("y: ");

    int z = add(x, y);
    printf("summ = %i\n", z);

    float d =  (float) x / (float) y;
    printf("division = %.20f\n", d);
}

int add(int a, int b)
{
    return a + b;
}

File: week06_python/lecture06/calculator.py
def main():
    x = get_int("x: ")
    y = get_int("y: ")

    z = x / y

    print(f"division: {z}")


def get_int(prompt):
    while True:
        try:
            return int(input(prompt))
        except ValueError:
            # pass
            print("Not an integer")


main()
